libnavui-maps/src/main/java/com/mapbox/navigation/ui/maps/route/line/api/MapboxRouteLineView.kt

```diff
@@ -582,7 +582,7 @@ class MapboxRouteLineView(var options: MapboxRouteLineOptions) {
 
     private fun updateSource(style: Style, sourceId: String, featureCollection: FeatureCollection) {
         style.getSource(sourceId)?.let {
-            (it as GeoJsonSource).featureCollection(featureCollection)
+            (it as GeoJsonSource).data(featureCollection.toJson())
```

Comment:
`toJson` is a heavy operation and cannot happen synchronously here. We'd probably should do it in the `MapboxRouteLineApi` on a worker thread and deliver as an internal field in `RouteSetValue`.

Reply:
Capturing from a internal chat, this is not needed at all, we can use the regular feature collection as we did!

Reply:
done